source/ref_installation.md

```diff
@@ -0,0 +1,415 @@
+# ODAHU installation from scratch
+
+## 1. Kubernetes setup
+
+minimum version is 1.13
```

Comment:
Convert this into a sentence.

Reply:
Done in c35c683823bc4bedc40272cafa686519adcc5856.